build-ffmpeg

```diff
@@ -679,9 +678,43 @@ download "https://ffmpeg.org/releases/ffmpeg-4.3.1.tar.bz2"
 	--extra-libs="${EXTRALIBS}" \
 	--pkgconfigdir="$WORKSPACE/lib/pkgconfig" \
 	--pkg-config-flags="--static" \
-	--prefix="${WORKSPACE}"
+	--prefix="${WORKSPACE}" \
+	--disable-stripping \
+	--enable-debug
 
 execute make -j $MJOBS
+
+if is_mac; then
+	execute dsymutil libavcodec/libavcodec.dylib -o libavcodec.dSYM
+	execute dsymutil libavdevice/libavdevice.dylib -o libavdevice.dSYM
+	execute dsymutil libavfilter/libavfilter.dylib -o libavfilter.dSYM
+	execute dsymutil libavformat/libavformat.dylib -o libavformat.dSYM
+	# we're not building this one
+	# execute dsymutil libavresample/libavresample.dylib -o libavresample.dSYM
+	execute dsymutil libavutil/libavutil.dylib -o libavutil.dSYM
+	execute dsymutil libpostproc/libpostproc.dylib -o libpostproc.dSYM
+	execute dsymutil libswresample/libswresample.dylib -o libswresample.dSYM
+	execute dsymutil libswscale/libswscale.dylib -o libswscale.dSYM
+fi
+
+if [[ -n "$SENTRY_AUTH_TOKEN" ]]; then
+	# TODO: install sentry-cli when running on CI
+	sentry-cli --auth-token $SENTRY_AUTH_TOKEN upload-dif --org descript-inc --project descript-electron .
```

Comment:
is there a way we can include these dSYM files somewhere so this repo doesn't need to do the uploading itself?

Reply:
1. I guess the question is: where would we want to do that? None of the options really "make sense" aside from in beachcube itself. But it seems weird to carry all of the dSYMs from this twice-removed library all the way to that repo.
2. We're uploading these assets (dSYMS + dylibs) to Sentry before stripping them. The dSYMs and dylibs actually have different debug properties (sentry-cli reports that one has `symtab` and `unwind` while the other has `symtab` and `debug`) so we want both (pre-stripping). So we'd have to publish the unstripped binaries somewhere if we didn't want to upload here.

Reply:
makes sense. then let's make the org and project environment variables then, so it's not descript-specific logic